src/components/__tests__/Button.test.tsx

```diff
@@ -53,7 +53,7 @@ describe('Button', () => {
               class="reactist_popover__wrapper reactist_tooltip__wrapper"
             >
               <button
-                class="reactist_button reactist_button--primary"
+                class="reactist_button reactist_button reactist_button--primary reactist_button--primary"
```

Comment:
What's happening here? Why do we add all classes twice?

Reply:
I think it is because we still add `reactist_button` for backward compatibility, but we also add `styles['reactist_button']` from the css module. It so happens that in tests css modules are most probably being mocked and it just returns the same name as the class name, instead of a dynamically generated one.

This is not the reason why asked for it, but it also surfaces the fact that there's no need to call the css module class names the way we did, as they're now local to the button. For instance, `reactist_button` can become `container` and `reactist_button--primary` can become `primary`.

Reply:
Sounds good, please do change the new class names to something else @vishnugopal 👍